sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/interface.scala

```diff
@@ -93,12 +93,16 @@ object CatalogStorageFormat {
  * @param spec partition spec values indexed by column name
  * @param storage storage format of the partition
  * @param parameters some parameters for the partition
+ * @param createTime creation time of the partition
```

Comment:
let's mention the time unit, i.e. in milliseconds.

Reply:
OK, it's the same as CatalogTable, in milliseconds. I fill fix this comment.